registry/tracecat_registry/integrations/microsoft_graph.py

```diff
@@ -122,3 +121,202 @@ async def send_teams_message(
         response = await client.post(url, headers=headers, json=payload)
         response.raise_for_status()
         return response.json()
+
+
+@registry.register(
+    default_title="Create Teams channel",
+    description="Create a new channel in a Microsoft Teams team. Can create either public (standard) or private channels.",
+    display_group="Microsoft Graph",
+    doc_url="https://learn.microsoft.com/en-us/graph/api/channel-post",
+    namespace="tools.microsoft_graph",
+    secrets=[microsoft_oauth_secret],
+)
+async def create_teams_channel(
+    team_id: Annotated[
+        str, Field(..., description="The ID of the team to create the channel in.")
```

Comment:
prefer to use `Doc(...)` from `typing_extensions` over `Field()` as its simpler. should replace this for all